Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -416,8 +416,15 @@ private static Ray CalculateProjectedSafeZoneRay(Vector3 origin, IMixedRealityCo
                 default:
                 case SolverSafeZone.UlnarSide:
                     {
-                        direction = Vector3.Cross(CameraCache.Main.transform.forward, Vector3.up);
-                        direction = IsPalmFacingCamera(hand) ? direction : -direction;
+                        if (rotationBehavior == SolverRotationBehavior.LookAtTrackedObject)
+                        {
+                            direction = targetTransform.right;
+                        }
+                        else
```

Comment:
i feel like here and below should be 
`else if (rotationBehavior == SolverRotationBehavior.LookAtMainCamera)`
?
not sure yet what result should be when SolverRotationBehavior.None?

Reply:
@Troy-Ferrell I added another enum here called SolverOffsetBehavior for clarity. There is no none option here so this is no longer an issue. I believe the additional enum adds both clarity and options for users.